api/src/integrations/core-documents/base-record-utils.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const defaultLog = require('../../utils/logger')('core-base-record-utils');
+const RecordController = require('../../controllers/record-controller');
+
+/**
+ * Core base record type handler that can be used directly, or extended if customizations are needed.
+ *
+ * Must contain the following functions:
+ * - transformRecord: (object) => object
+ * - saveRecord: (object) => any
```

Comment:
Nit: could delete.

This used to be true in the early days of the epic importer, but isnt the case ever since I moved it to more of a class based approach with overriding functions, etc.  Probably I missed removing it then, and its just propagated understandably.

Reply:
Removed